ui-tests/src/test/java/org/openmrs/reference/page/CaptureVitalsPage.java

```diff
@@ -21,14 +22,13 @@ public String getPageUrl() {
         return "/coreapps/findpatient/findPatient.page?app=referenceapplication.vitals";
     }
 
-//    public void search(String text) {
-//        WebElement searchField = findElement(PATIENT_SEARCH);
-//        try {
-//            searchField.sendKeys(text);
-//        } catch(Exception e) {
-//            e.printStackTrace();
-//        }
-//        clickOn(FIRST_FOUND_PATIENT);
-//    }
-
+    public void search(String text) {
```

Comment:
Do we need this method at all? Why is this in this PR?

Reply:
> Do we need this method at all? Why is this in this PR?

I actually think we need this but will investigate what it functions.

Reply:
I don't disagree that we might want it, I'm more of wondering why it's in this PR. It's better to keep a PR to doing the minimum amount necessary to accomplish what the ticket asks. One reason for this is that if this PR causes issues, we can just revert a single commit and fix things whereas if we have multiple things happening in that commit, reverting that commit will unexpectedly break other things.

Reply:
Hey @ibacher . we really dont need this functionality at all.On capturing vitals No where we are searching for patient